woocommerce/wc-admin
# Products table

Fixes #535.

Adds a table connected to the API to the _Products_ report.

As explained in #535, there is some data which is still not returned by the API, for now it's replaced with empty defaults and is marked with `// @TODO` comments in the code.

### Screenshots
![image](https://user-images.githubusercontent.com/3616980/47292359-f7c79480-d606-11e8-9b27-c50d6b20cd64.png)

### Detailed test instructions:

* Go to _Analytics_ > _Products_.
* Verify the table with products is there and works as expected.
* Try sorting rows, downloading its contents, clicking on the links, etc.
